src/command/FileCommands.cxx

```diff
@@ -127,12 +127,7 @@ gcc_pure
 static bool
 IsValidValue(const StringView s) noexcept
 {
-	for (const char ch : s) {
-		if ((unsigned char)ch < 0x20)
-			return false;
-	}
-
-	return true;
+	return std::all_of(s.begin(), s.end(), [](const auto &ch) { return (unsigned char)ch < 0x20; });
```

Comment:
This is inverted, just in a different way than before....
Previously, it required that zero elements are below `0x20`.
Now it requires that *all* elements are below `0x20`.